recipes/pytfa/meta.yaml

```diff
@@ -0,0 +1,59 @@
+{% set name = "pytfa" %}
+{% set version = "0.9.3" %}
+{% set build_number = '0' %}
+
+package:
+  name: "{{ name|lower }}"
+  version: "{{ version }}"
+
+source:
+  url: https://github.com/EPFL-LCSB/{{ name }}/archive/refs/tags/{{ version }}.tar.gz
+  sha256: b4d848bd04cf8384b05e82fecb25a7cc59a65031ac5dee2c732853a12b086014
+
+build:
+  number: 0
+  noarch: python
+  script: "{{ PYTHON }} -m pip install . -vv"
+
+requirements:
+  host:
+    - pip
+    - python >=3.8
+
+  run:
+    - python >=3.8
+    - equilibrator-api
+    - equilibrator-cache
+    - sympy <=1.5
+    - networkx
+    - pandas
+    - cobra <=0.24.0
+    - bokeh >=0.12.1
+    - numpy
+    - optlang
+    - scipy
+    - tqdm
+    - python-libsbml
+
+test:
+  imports:
+    - pytfa
+  commands:
+    - pip check
+  requires:
+    - pip
+    - pytest
```

Comment:
Please report this upstream. It is very unlikely for a package to require pytest in its `install_requires` section. pytest should be listed in an extras/optional section.

Reply:
Sure will do